aptly/init.sls

```diff
@@ -13,22 +15,17 @@ aptly:
   pkg.installed:
     - refresh: True
     - pkgs:
-      - aptly
-      - gnupg1
-      - gpgv1
-      - bzip2
+{% for item in aptly.pkgs %}
```

Comment:
You might literally be able to just something like {{ aptly.pkgs | toyaml | indent 4 }} here.  (just guessing from memory - grep for 'indent' in other formulas.